draft-ietf-quic-manageability.md

```diff
@@ -945,68 +945,72 @@ deliver packets between server and NAT. Reducing the timeout on UDP NATs might
 be tempting in light of this property, but not all QUIC server deployments will
 be robust to rebinding.
 
+Recommendations on time-outs for stateful network functions such as NATs is
+provided in {#sec-stateful}.
+
 ### Resource Conservation
 
-NATs sometimes hit an operational limit where they exhaust available public IP
-addresses and ports, and must evict flows from their address/port mapping. CIDs
-might appear to offer a way to multiplex many connections over a single
-address and port.
-
-However, QUIC endpoints may negotiate new connection IDs inside
-cryptographically protected packets, and begin using them at will. Imagine two
-clients behind a NAT that are sharing the same public IP address and port. The
-NAT is differentiating them using the incoming Connection ID. If one client
-secretly changes its connection ID, there will be no mapping for the NAT, and
-the connection will suddenly break.
-
-QUIC is deliberately designed to fail rather than persist when the network
-cannot support its operation. For HTTP/3, this extends to recommending a
-fallback to TCP-based versions of HTTP rather than persisting with a QUIC
-connection that might be unstable. And {{?I-D.ietf-quic-applicability}}
-recommends TCP fallback for other protocols on the basis that this is preferable
-to sudden connection errors and time outs.
+Use of QUIC's connection ID for NAT mappings is not a viable approach to
+extend operational limits of deployed NATs, as the connection ID is not
+guaranteed to be constant over the lifetime of a connection. Endpoints may 
+begin using new connection IDs at will at any time during the connection.
+As such, the use of the connection ID
+to multiplex connections over a single address and port would cause the
+connectivity to break when the connection ID changes to a value that does not have
+an active mapping.
+
+Not-yet-used connection IDs are unknown to in-network devices and cannot
+be easily associated with a connection ID in use, as new connection IDs are
+negotiated inside cryptographically protected packets between the endpoints
+and as such are not observable in the network. Alternatively, sharing
+configurations between the client and NAT might be logistically challenging
+or impossible, especially when client migrates with an existing connection to
+a point behind the NAT. {{QUIC_LB}} described mechanisms to encode the client's
+identity in
+a connection ID to be used for load balancing. However, the described approach
+requires explicit coordination between the endpoint and network devices using the
+connection ID under the assumption that servers and load balancers are deployed
+by the same entity or cooperating entities.
```

Comment:
This might be replaced by text that just says "Coordination with endpoints is necessary if connection IDs are to be used as QUIC provides confidentiality protection for the negotiation of connection IDs.  A NAT might be able to coordinate with endpoints using the methods described in {{QUIC_LB}}."